src/decision/src/VisionDecision.cpp

```diff
@@ -1,15 +1,21 @@
 /*
- * Created By: Gareth Ellis
+ * Created By: Robyn Castro
  * Created On: September 22, 2016
  * Description: The vision decision node, takes in an image from the robot's
  *              camera and produces a recommended twist message
  */
-
 #include <VisionDecision.h>
+#include <tiff.h>
+
+#define PI 3.14159265
+#define null -1
+#define INVALID 91
+#define NOISEMAX 30
```

Comment:
I sort of get what this is for, but it could probably use a comment preceding it explaining exactly what it is and what it's used for.